manager/src/main/resources/db/migration/0.1/V0.1.16__add_quartz_scheduler.sql

```diff
@@ -0,0 +1,213 @@
+DROP TABLE IF EXISTS QRTZ_FIRED_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_PAUSED_TRIGGER_GRPS;
+DROP TABLE IF EXISTS QRTZ_SCHEDULER_STATE;
+DROP TABLE IF EXISTS QRTZ_LOCKS;
+DROP TABLE IF EXISTS QRTZ_SIMPLE_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_CRON_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_SIMPROP_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_BLOB_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_JOB_DETAILS;
+DROP TABLE IF EXISTS QRTZ_CALENDARS;
+DROP TABLE IF EXISTS TASKS;
+DROP SEQUENCE IF EXISTS hibernate_sequence;
+
+CREATE SEQUENCE hibernate_sequence start 1 increment 1;
+
+CREATE TABLE TASKS
+(
+    id        int8 NOT NULL,
+    createdAt TIMESTAMP,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE QRTZ_JOB_DETAILS
```

Comment:
Are these SQL commands taken from some specific external location?
Are they supposed to change in some future Quartz version or will remain the same?

Reply:
It was lifted from [Quarkus](https://github.com/quarkusio/quarkus-quickstarts/blob/main/quartz-quickstart/src/main/resources/db/migration/V2.0.0__QuarkusQuartzTasks.sql) (linked from [here](https://quarkus.io/guides/quartz)) which seems to follow the [Quartz](https://github.com/quartz-scheduler/quartz/blob/master/quartz-core/src/main/resources/org/quartz/impl/jdbcjobstore/tables_postgres.sql) DDL.

I have however now added the Quartz DDL (as I notice Quarkus added an additional table).

As to whether they change.. they could and it is a risk; but I'd assume it's the same for any consumer of Quartz with a database backed `JobStore` (Quartz's storage abstraction) and something we'd need to cover. Our integration tests would fail if a Quartz/Quarkus version change broke the DDL. Our Unit Tests use an in-memory `JobStore` and hence are not sensitive to DDL issues.

Reply:
ok, thanks for info